nibabel/imagestats.py

```diff
@@ -0,0 +1,62 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""
+Functions for computing image statistics
+"""
+
+import numpy as np
+from nibabel.imageclasses import spatial_axes_first
+
+def count_nonzero_voxels(img):
+    """
+    Count number of non-zero voxels
+    Parameters
+    ----------
+    img : ``SpatialImage``
+        All voxels of the mask should be of value 1, background should have value 0.
+
+    Returns
+    -------
+    non zero voxel volume: int
```

Comment:
Simple variable names are fine.

```suggestion
    count : int
```